src/components/manifold-oauth/manifold-oauth.tsx

```diff
@@ -1,35 +1,57 @@
-import { Event, EventEmitter, Component, h, Prop } from "@stencil/core";
-import { AuthToken, PumaAuthToken } from "../../interface";
+import { Event, EventEmitter, Component, h, Prop } from '@stencil/core';
+import { AuthToken, PumaAuthToken } from '../../interface';
 
-@Component({
-  tag: "manifold-oauth"
-})
+@Component({ tag: 'manifold-oauth' })
 export class ManifoldOauth {
-  @Prop() oauthUrl?: string = "https://login.manifold.co/signin/oauth/web";
+  @Prop() oauthUrl?: string = 'https://login.manifold.co/signin/oauth/web';
   @Event() receiveManifoldToken: EventEmitter<AuthToken>;
 
-  private loadTime?: Date;
+  private loadTime?;
```

Comment:
Why was the type removed?

Reply:
Oh that was an ESLint rule suggestion from @typescript-eslint. I can revert it for now and fix it in the ESLint PR